app/internal/out_of_office.py

```diff
@@ -0,0 +1,93 @@
+from datetime import datetime
+from sqlalchemy.orm import Session
+
+from app.database.models import User, OutOfOffice
+
+
+def get_who_is_out_of_office(session: Session,
+                             event_start_date,
```

Comment:
please add types

Reply:
Done! Thanks :)

Reply:
I don't see it.. :) Did you push the changes?